data.sql

```diff
@@ -7,3 +7,7 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ( 'Pikachu', '07-01-2021', 1, FALSE, 15.04);
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ( 'Devimon', '12-5-2020', 5, TRUE, 11);
+
+ALTER TABLE animals ADD species VARCHAR;
+
```

Comment:
- [ ] Everything that creates or change the tables structure should be saved in the schema.sql file. Please move the `ALTER` statement form data.sql to schema.sql file.